src/com/xilinx/rapidwright/design/ModuleInst.java

```diff
@@ -383,12 +404,13 @@ public boolean place(Site newAnchorSite, boolean skipIncompatible, boolean allow
                         return false;
                     }
                 }
-                PIP newPip = new PIP(pip);///new PIP(newPipTile, pip.getStartWire(), pip.getEndWire(), pip.getPIPType());
+                PIP newPip = new PIP(pip);
                 newPip.setTile(newPipTile);
-                //if (!newPipTile.hasPIP(newPip)) {
-                //    return false;
-                //}
-                net.addPIP(newPip);
+                if (pipSet != null) {
+                    pipSet.add(newPip);
+                } else {
+                    net.addPIP(newPip);
+                }
```

Comment:
Wouldn't you always want to insert this PIP into the global net...
```suggestion
                }
                net.addPIP(newPip);
```